api/applications/applications_handler.go

```diff
@@ -270,6 +271,19 @@ func (ah ApplicationHandler) ModifyRegistrationDetails(appName string, patchRequ
 		runUpdate = true
 	}
 
+	if patchRequest.ConfigBranch != nil && *patchRequest.ConfigBranch != "" {
```

Comment:
I suggest we start always trim spaces from requested strings

Reply:
Agreed. I'll fix it

Reply:
Implemented your suggestions